src/constants.ts

```diff
@@ -91,36 +90,13 @@ export const QUESTIONAIRRE: inquirer.QuestionCollection = [
     }
   },
 
-  // TESTING BACKEND
-  {
-    type: 'list',
-    name: 'backend',
-    message: 'Where do you want to run your e2e tests?',
-    choices: [
-      {name: 'On my local machine', value: 'local'},
-      {name: 'On a remote machine (cloud)', value: 'remote'},
-      {name: 'Both', value: 'both'}
-    ],
-    default: 'local'
-  },
 
-  {
-    type: 'list',
-    name: 'cloudProvider',
-    message: '(Remote) Please select your cloud provider:',
-    choices: [
-      {name: 'BrowserStack', value: 'browserstack'},
-      {name: 'Sauce Labs', value: 'saucelabs'},
-      {name: 'Other providers or remote selenium-server', value: 'other'}
-    ],
-    when: (answers) => ['remote', 'both'].includes(answers.backend)
-  },
 
   // BROWSERS
   {
     type: 'checkbox',
     name: 'browsers',
-    message: 'Which browsers will you be testing on?',
+    message: 'Select target browsers',
     choices: (answers) => {
       let browsers = BROWSER_CHOICES;
       if (answers.backend === 'local' && process.platform !== 'darwin') {
```

Comment:
`answers.backend` will not be applicable here, since the backend question has been moved to the bottom.